overview/content-types/shell-code.nix

```diff
@@ -6,14 +6,37 @@
 }:
 let
   inherit (lib) mkOption types;
+
+  codeBlock.one =
```

Comment:
This can be a module field, simply a string value that reads from config. or in a `let` block at `__toString`